### PR TITLE
[Do not merge] rename parameter

### PR DESCRIPTION
**Pull request by fmessmer**
parameter renaming in https://github.com/ipa320/cob_control/pull/133

### CODE REVIEW

**Review by floweisshardt — DISMISSED** [review]

**Comment by fmessmer** [comment]
Do not merge without https://github.com/ipa320/cob_control/pull/133

File: cob_hardware_config/cob4-2/config/velocity_smoother.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ accel_lim_w: 0.5
 # Optional parameters
 frequency: 50.0
 decel_factor: 1.5       #used if zero velocity is received or goal velocity is far away from current velocity
-decel_factor_safe: 2.5  #used if no velocity commands received (eg. no deadman on joystick)
+safe_factor: 2.5        #used if no velocity commands received (eg. no deadman on joystick)
```
Comment (benmaidel) [inline cob_hardware_config/cob4-2/config/velocity_smoother.yaml:17]: Why the change? This factor should only be applied on deceleration.

**Review by benmaidel — CHANGES_REQUESTED** [review]

**Comment by ipa-rmb** [comment]
Is this still relevant or can be closed?

**Comment by fmessmer** [comment]
not in the focus anymore

**Pull request closed without merging**